tests/test_views.py

```diff
@@ -4,6 +4,7 @@ def test_view(client):
     assert not resp.is_json
     assert resp.mimetype == "text/html"
     assert "<title>Celery Director</title>" in str(resp.data)
+    assert 'const REPO_LINK="https://github.com/ovh/celery-director"' in str(resp.data)
```

Comment:
Same comment as `self.REPO_LINK` in the `settings.py` file.

Reply:
see  answer to https://github.com/ovh/celery-director/pull/141/files/3d2c0bfcac90c9740b84d34fb3514fd3f175951b#r849687363